components/product/single.js

```diff
@@ -23,12 +35,30 @@ export const SingleProductComponent = ({
           className="px-3 py-2 bg-indigo-200 text-gray-800"
           onClick={(e) => {
             e.preventDefault();
-            onAddToCartClick(product);
+             onAddToCartClick(product);             
```

Comment:
this is a bit weird 1 space extra indentation.

Reply:
Sorry i didn't notice those error..